readiness/readiness.go

```diff
@@ -23,8 +23,12 @@ func CheckIfDeploymentsAreReady() error {
 		return err
 	}
 
+	dashboardSelector := "app=" + data.AppConfig.DashboardDeploymentName
```

Comment:
Please replace the raw string with [`LabelSelector` ](https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#LabelSelector) type.

For example:

```go
	// reference: https://github.com/kubernetes/apimachinery/issues/47#issuecomment-431062847
labelSelector := v1.LabelSelector{
	MatchLabels: map[string]string{"app": "data.AppConfig.DashboardDeploymentName"},
}
pods, err := clientset.CoreV1().Pods("ns").List(
	context.TODO(),
	v1.ListOptions{LabelSelector: labels.Set(labelSelector.MatchLabels).String()},
)
```